deploy/00_deploy_seed_contracts.js

```diff
@@ -26,6 +28,19 @@ const deployFunction = async ({ getNamedAccounts, deployments, network }) => {
     args: [Safe, seedFactoryAddress],
     log: true,
   });
+
+  const signerInstance = await ethers.getContract("Signer");
+  const seedInstance = await ethers.getContract("Seed");
```

Comment:
Just cosmetic details: You can use the seedAddress from line 16, no need to instantiate again. And you could do the same as in line 16 also in line 26. Then you would not need to instantiate the Signer as well.